components/shared/BetSlipCard/index.tsx

```diff
@@ -0,0 +1,100 @@
+'use client';
+
+import BitTorentCoinIcon from '@/components/icons/BitTorentCoinIcon';
+import LogoWithText from '../LogoWithText';
+import ArrowDownIcon from '@/components/icons/ArrowDownIcon';
+import XIcon from '@/components/icons/XIcon';
+import { Listbox, Transition } from '@headlessui/react';
+import { useState, Fragment } from 'react';
+
+const bids = [5, 10, 20, 30, 50, 100];
+
+function BetSlipCard() {
+  const [selected, setSelected] = useState<number>(bids[0]);
+
+  return (
+    <div className="w-full bg-xport-gray-alternate text-xs rounded">
```

Comment:
this should be width full, by my view this is not full width??? enough??
<img width="491" alt="Screen Shot 2023-09-20 at 18 00 15" src="https://github.com/galihpermana29/x-sports/assets/54941568/e6d49ec7-915d-41d1-9c6c-4ab93ad3fd51">

also the payout shouldbe automatically calculated if the dropdown is change... by the calculation payout = dropdown value * odds

Reply:
I added max-w-screen-xl which is around 1280px to maintain the ratio on bigger screen. Should I make it to 2xl (1536px) or just remove the max width?